features/configuration/configuration_controller.rb

```diff
@@ -73,14 +73,7 @@ class ConfigurationController < AuthenticatedControllerBase
 
     # @return [Hash]
     def keys
-      {
-        encryption_key: ENV["FASTLANE_CI_ENCRYPTION_KEY"],
-        ci_user_email: ENV["FASTLANE_CI_USER"],
-        ci_user_api_token: ENV["FASTLANE_CI_PASSWORD"],
-        repo_url: ENV["FASTLANE_CI_REPO_URL"],
-        clone_user_email: ENV["FASTLANE_CI_INITIAL_CLONE_EMAIL"],
-        clone_user_api_token: ENV["FASTLANE_CI_INITIAL_CLONE_API_TOKEN"]
-      }
+      FastlaneCI.env.all
```

Comment:
Eventually we probably want to white-list some of them, instead of just exposing all I'd think?

Reply:
What do you mean white-list some of them?

Reply:
Similar to how Rails handles parameters, instead of just taking all the `POST` parameters and trying to map it to a model, you define a list of allowed parameters, if they're there, good, if not also fine.

So instead of saying `.env.all`, you'd still have a list of env variables that we allow setting and try to map here, but haven't thought this through. actually feel free to ignore my comments here